ARCHITECTURE.md

```diff
@@ -0,0 +1,113 @@
+# DSO Toolkit architecture
+
+Dit document beschrijft de visie en architectuur van het DSO Toolkit landschap.
+
+Het begint met een samenvatting waarna per onderdeel een uitgebreide beschrijving volgt.
+
+## Synopsis
+
+De DSO Toolkit is een Design System en bestaat uit de volgende NPM packages:
+
+* `@dso-toolkit/sources`
+* `dso-toolkit`
+* `@dso-toolkit/css`
+* `@dso-toolkit/core`
+* `@dso-toolkit/react`
+* `@dso-toolkit/leaflet`
+* `@dso-toolkit/react-leaflet`
+
+![DSO Toolkit dependency graph](dependency-graph.svg)
+
+## Termen
+
+* **implementatie**: Binnen de context van de toolkit bedoelen we hier een *type uitvoering*: Een component heeft één of meerdere implementaties: HTML/CSS component, Web Component, React Component, Angular Component. Buiten de context van de toolkit kan een afnemer van de toolkit een component binnen zijn applicatie implementeren.
+* **maintainer**: Een maintainer beheert de codebase van de DSO Toolkit.
+* **component**: De DSO Toolkit maakt geen gebruik van een methode zoals Atomic Design: Binnen de toolkit is alles een component. Van een heading zoals `h1` tot een complexe werkvorm zoals Accordion.
```

Comment:
Is 'composiet' of 'compositie' ook nog iets dan benoemd zou moeten worden?

Reply:
> * **composiet/compositie**: De DSO Toolkit levert componenten die als bouwstenen kunnen worden ingezet. Soms is het door middel van een markup voorschrift (de afnemer maakt de composiet) maar maintainers kunnen ook composieten maken: met Sass mixins en Web Components.

Wat denk je hiervan?

Reply:
Of...
**composiet/compositie**: De DSO Toolkit levert componenten die als bouwstenen kunnen worden ingezet.
Als een combinatie van bouwstenen eigen styling ontvangt, en als zodanig gedocumenteerd wordt, is het een composiet.

Reply:
(of is dat niet zo, wordt het daarmee een nieuw component)

Reply:
We laten het achterwege.